spec/controllers/corporate_controller_spec.rb

```diff
@@ -61,4 +64,41 @@
       end
     end
   end
+
+  describe 'POST create' do
+    let(:tool) { instance_double(Core::Article, id: "#{valid_partner[:tool_name].downcase.strip.gsub(' ', '-')}-syndication", categories: [syndication_category]) }
+    let(:valid_partner)   { FactoryGirl.attributes_for(:corporate_partner) }
+    let(:invalid_partner) { { name: 2323, width: 'sasd' } }
+
+    context 'with valid attributes' do
+      it 'creates a new partner' do
+        expect{
```

Comment:
rubocop would warn on whitespace and to use a do .. end instead of curly braces here